File: Fifty-Two Frames/FTFListLayout.m
//
//  FTFListLayout.m
//  FiftyTwoFrames
//
//  Created by Gershy Lev on 3/15/16.
//  Copyright © 2016 Gershy Lev. All rights reserved.
//

#import "FTFListLayout.h"

@implementation FTFListLayout

- (instancetype)init {
    if (self = [super init]) {
        [self setup];
    }
    
    return self;
}

- (void)setup {
    double width = [UIScreen mainScreen].bounds.size.width;
    self.itemSize = CGSizeMake(width, width + 27);
//    self.sectionInset = UIEdgeInsetsMake(0, 6, 0, 6);
//    self.minimumInteritemSpacing = 8.0;
//    self.minimumLineSpacing = 8.0;
}

- (void)prepareLayout {
    [super prepareLayout];
    [self setup];
}

@end

File: Fifty-Two Frames/FTFListLayout.swift
//
//  FTFListLayout.swift
//  FiftyTwoFrames
//
//  Created by Josh Siegel on 3/12/16.
//  Copyright © 2016 Gershy Lev. All rights reserved.
//

import UIKit

@objc class FTFListLayout: UICollectionViewFlowLayout {
    var numberOfColumns: CGFloat = 1.0
    
    override init() {
        super.init()
        setup()
    }
    
    required init?(coder aDecoder: NSCoder) {
        super.init(coder: aDecoder)
        setup()

    }
    
    func setup() {
        let width = (UIScreen.mainScreen().bounds.size.width) / numberOfColumns
        self.itemSize = CGSizeMake(width, width / 1.32)
    }
    
    override func prepareLayout() {
        setup()
    }
}